app/views/publish_document/published.html.erb

```diff
@@ -26,11 +26,16 @@
 
       <%= render "govuk_publishing_components/components/govspeak" do %>
         <%= govspeak_to_html t("publish_document.published.published_without_review.body_govspeak", title: @document.title) %>
+        <% stripped_url = DocumentUrl.new(@document).public_url.sub(/^https?\:\/\//,'') %>
```

Comment:
Nice 👍 - Do we need to do the same thing with the URL above too for when it doesn't need a review? If so might want to have a helper method for this

Reply:
I think you're right, we can do the same to both URls. Would you put that method in the DocumentUrl service or the view? Or maybe in a helper?

Reply:
Seems sufficiently generic that it can live in the application helper

Reply:
Done